{"name": "test_login_0_failed_wrong_datas[data1]", "status": "passed", "attachments": [{"name": "log", "source": "1956f05d-1457-4e6c-aaa1-522cdfd00b12-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'user': '', 'passwd': 'python', 'check': '请输入手机号'}"}], "start": 1608298461660, "stop": 1608298461939, "uuid": "11f863cc-87c6-4b1e-8ea9-a91d0ba7b4c0", "historyId": "50b62d68a5c78b16f277d604e09ea628", "testCaseId": "735933ecd585cb26d239d15346a80fa1", "fullName": "testcase.test_1login_pytest.TestLogin#test_login_0_failed_wrong_datas", "labels": [{"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_1login_pytest"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-J6CF24D"}, {"name": "thread", "value": "59848-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_1login_pytest"}]}